source/main/physics/BeamFactory.cpp

```diff
@@ -164,39 +100,31 @@ ActorManager::ActorManager()
 
     if (RoR::App::app_multithread.GetActive())
     {
-        // Create thread pool
-        int numThreadsInPool = ISETTING("NumThreadsInThreadPool", 0);
+        // Create worker thread (used for physics calculations)
+        m_sim_thread_pool = std::unique_ptr<ThreadPool>(new ThreadPool(1));
 
-        if (numThreadsInPool > 1)
+        // Create general-purpose thread pool
+        if (!BSETTING("DisableThreadPool", false))

```

Comment:
`app_multithread` determines whether or not the game runs single threaded.
`DisableThreadPool` allows you to run multi threaded, but without the general purpose threadpool. I think it should stay like that for now.

Reply:
> It would be great if we could change the amount of multi-threading while the game is running - for testing purposes.

I'm pursuing precisely this goal with my GVars - once I remake the ingame console with DearIMGUI, I'll make all GVars adjustable from there (live, using the Pending mechanism) on user's own risk. That's why I suggested adding `GVarPod_APS<int> app_threadpool_size` below.

Reply:
I really need to take a closer look at the new GVar system.